### PR TITLE
feat: add https://github.com/szkiba/xk6-cache extension

### PR DESCRIPTION
**Pull request by szkiba**
[xk6-cache](https://github.com/szkiba/xk6-cache) enables vendoring [Remote HTTP(s) modules](https://k6.io/docs/using-k6/modules#remote-http-s-modules) to single source control friendly file.

### CODE REVIEW

**Comment by szkiba** [comment]
Sorry about so much trouble on formatting...

**Comment by szkiba** [comment]
Hi @imiric, @na-- ,  could you plz help to merge this PR? (all checks are ok)

File: src/data/ecosystem/extensions.js [hunk 1]
```diff
@@ -420,6 +420,19 @@ const extensions = [
     official: false,
     categories: ['Reporting', 'Observability'],
   },
+  {
+    name: 'xk6-cache',
+    description:
+      'A k6 extension enables vendoring remote HTTP modules to single source control friendly file.',
```
Comment (imiric) [inline src/data/ecosystem/extensions.js:426]:
```suggestion
      'A k6 extension that enables vendoring remote HTTP modules to a single source control friendly file.',
```

**Review by imiric — APPROVED** [review]
Thanks for this and for the extension! Just left a minor nitpick to reword the description.

**Comment by szkiba** [comment]
> Thanks for this and for the extension! Just left a minor nitpick to reword the description.

@imiric thank you, committed

**Comment by szkiba** [comment]
@imiric thank you for approve

**Review by na-- — APPROVED** [review]

**Pull request merged**